File: net/retrieving-linked-content/AuthorModel.cs
// DocSection: linked_content_author_model
// Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models

using System;
using System.Collections.Generic;
using Kentico.Kontent.Delivery;

namespace KenticoKontentModels
{
    public partial class Author
    {
        public const string Codename = "author";
        public const string NameCodename = "name";
        public const string BioCodename = "bio";

        public string Name { get; set; }
        public IRichTextContent Bio { get; set; }
    }
}
// EndDocSection

File: ios/retrieve-linked-content/AuthorModel.swift
// DocSection: linked_content_author_model
// Create strongly typed models according to https://kontent.ai/learn/strongly-typed-models
// Tip: Find more about Swift SDK at https://kontent.ai/learn/ios

import ObjectMapper
import KenticoKontentDelivery

public class Author: Mappable {
    var name: TextElement?
    var bio: RichTextElement?
    
    public required init?(map: Map){
        let mapper = MapElement.init(map: map)
        name = mapper.map(elementName: "name", elementType: TextElement.self)
        bio = mapper.map(elementName: "bio", elementType: RichTextElement.self)
    }
    
    public func mapping(map: Map) {
    }
}
// EndDocSection